<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmoteBuilder</class>
 <widget class="QMainWindow" name="EmoteBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmoteBuilder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0" columnstretch="1,0,0,0,0">
    <item row="0" column="4" rowspan="3">
     <layout class="QVBoxLayout" name="settingsPanel">
      <item>
       <layout class="QHBoxLayout" name="currentFrameField">
        <item>
         <widget class="QLabel" name="currentFrameLabel">
          <property name="text">
           <string>Current Frame Number: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentFrameInput">
          <property name="text">
           <string>0</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fpsField">
        <item>
         <widget class="QLabel" name="fpsLabel">
          <property name="text">
           <string>FPS: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fpsInput">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="frameSetPanel">
        <item>
         <widget class="QPushButton" name="prevFrameButton">
          <property name="text">
           <string>Previous Frame</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextFrameButton">
          <property name="text">
           <string>Next Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="playStopPanel">
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="anchorPanel">
        <item>
         <widget class="QLabel" name="anchorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Anchor</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="anchorFields">
          <item>
           <widget class="QLabel" name="anchorXLabel">
            <property name="text">
             <string>X: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="anchorXInput">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="anchorYLabel">
            <property name="text">
             <string>Y: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="anchorYInput">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QPushButton" name="buildAtlasButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Build</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="viewLayout">
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="text">
         <string>Please load sprites from a folder to continue.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="spriteView">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="referenceView">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="emote_builder.qrc">:/Resources/Assets/Knight.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QPushButton" name="loadSpritesButton">
      <property name="text">
       <string>Load Sprites</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="emote_builder.qrc"/>
 </resources>
 <connections/>
</ui>
